### ISSUE TITLE
Displaying multiple values from the same metric within popup

### ISSUE THREAD

**Issue by pasquy73** [panodata/grafana-map-panel#28]
Hi all,
I would like to display in the popup, on the worldmap-panel, multiple attributes. For example:

```
New Jersey: 5 errors
items: 100
avg: 33 
```

So, my query can provide `name1`, `value1`, `name2`, `value2`, `lat`, `long`, `time_sec` for `location data` table and I want to display in the popup:
```
name1: value1
name2: value2
```
Is there a way to do this?
Thanks a lot.

**Comment by amotl** [comment]
Dear Pasquale,

thanks for writing in and for transferring this from the issue you created at https://github.com/grafana/worldmap-panel/issues/235 already.

We will be happy to consider getting into the details of this when working on the next development iteration. However, I currently can't promise about a time frame and will be traveling until the 7th of December or so as I also told @sekomarek, who just asked for #27.

Feel free to get back to me and bump this issue accordingly if you don't hear anything back.

With kind regards,
Andreas.

**Comment by pasquy73** [comment]
Hi Andreas,
thanks a lot. 
Don't worry, and enjoy your travels.

**Comment by pasquy73** [comment]
Hi Andreas,
it will be also interesting to add multiple contexts in the popup; for instance: 

```
New Jersey: 5 errors
items: 100
avg: 33 

New York: 8 errors
items: 171
avg: 42
sum: 1415 
```

**Comment by pasquy73** [comment]
> Dear Pasquale,
> 
> thanks for writing in and for transferring this from the issue you created at [grafana#235](https://github.com/grafana/worldmap-panel/issues/235) already.
> 
> We will be happy to consider getting into the details of this when working on the next development iteration. However, I currently can't promise about a time frame and will be traveling until the 7th of December or so as I also told @sekomarek, who just asked for #27.
> 
> Feel free to get back to me and bump this issue accordingly if you don't hear anything back.
> 
> With kind regards,
> Andreas.

Hi Andreas,
do you have any news about this topic? 
Thank you for your time.
Pasquale

**Comment by amotl** [comment]
Dear Pasquale,

when thinking about this, I wonder if your request isn't actually identical to https://github.com/grafana/worldmap-panel/issues/59? Actually, it is not only about **displaying** multiple metric values within the popup, but already **querying** them beforehand, right?

With kind regards,
Andreas.

**Comment by pasquy73** [comment]
Yes. I do a query to get **longitude**, **latitude**, **time_sec**, **name**, **value** fields and it works. 
I'm going to aggregate multiple values within the same _longitude_, _latitude_ in order to provide also _sum_, _average_, _numItems_ (value will be always the last value); so my new query provides **longitude**, **latitude**, **time_sec**, **name**, **value**, **sum**, **average**, **numItems** and I don't know how to show all params in the popup.

**Comment by amotl** [comment]
Ok, this is not **quite** **_multiple metrics_**, right? I am a bit unsure right now, so let me be safe to ask again specifically: Will there be more than one item configured on the metrics tab or will the fields be populated from within a single metric query?

**Comment by pasquy73** [comment]
Just a single metric query

**Comment by amotl** [comment]
> This is just about a single metric query.

Ok, thanks for clarifying this. We will track #43 seperately.

**Comment by hesam1992** [comment]
Hi guys.
I have the same problem too.

**Comment by vmanju123** [comment]
We have same problem, need to display multiple metrics on worldmap panel, single metric can drive the size of circle and color, also need to provide units for those metrics, For Eg: we have data bytes as metric it can easily run into trillions of bytes need decimal support and unti support for metrics as in other panels, Thanks

**Comment by amotl** [comment]
Dear @robouden,

within https://github.com/panodata/grafana-map-panel/issues/57#issuecomment-715894081, you asked:
> One more request. To get multiple values on the hover panel. That would be a big help for me and probably also other people.

Can you review this issue and confirm you are referring to the same feature request here?

With kind regards,
Andreas.

**Comment by robouden** [comment]
Andreas,

Not sure what you mean by "the same feature request here?".
I would like to see to have the options to display more than one variable being displayed in the rollover. Maybe that was a feature already asked for or not.

regards,
rob

**Comment by amotl** [comment]
As #77 will only improve this when using an Elasticsearch data source, I am deliberately reopening this in order to keep tracking it for other data sources.
